veto/veto.cxx

```diff
@@ -624,94 +754,94 @@ TGeoVolume* veto::MakeMagnetSegment(Int_t seg){
 	VertMagCover_Y->SetLineColor(15);
 	VertMagCover_X->SetLineColor(15);
 
-	if(isVert)tMagVol->AddNode(VertMagCover_Y, 1 , new TGeoCombiTrans(xPos,0,zPos,new TGeoRotation("r",0,0,0)));
-	if(isVert)tMagVol->AddNode(VertMagCover_Y, 2 , new TGeoCombiTrans(-xPos,0,zPos,new TGeoRotation("r",0,0,0)));
-	if(isVert)tMagVol->AddNode(VertMagCover_Y, 3 , new TGeoCombiTrans(xPos,0,-zPos,new TGeoRotation("r",0,0,0)));
-	if(isVert)tMagVol->AddNode(VertMagCover_Y, 4 , new TGeoCombiTrans(-xPos,0,-zPos,new TGeoRotation("r",0,0,0)));
+	if(isVert)tMagVol->AddNode(VertMagCover_Y, 1 , new TGeoCombiTrans(xpos,0,zpos,new TGeoRotation("r",0,0,0)));
+	if(isVert)tMagVol->AddNode(VertMagCover_Y, 2 , new TGeoCombiTrans(-xpos,0,zpos,new TGeoRotation("r",0,0,0)));
+	if(isVert)tMagVol->AddNode(VertMagCover_Y, 3 , new TGeoCombiTrans(xpos,0,-zpos,new TGeoRotation("r",0,0,0)));
+	if(isVert)tMagVol->AddNode(VertMagCover_Y, 4 , new TGeoCombiTrans(-xpos,0,-zpos,new TGeoRotation("r",0,0,0)));
 
-	if(isVert)tMagVol->AddNode(VertMagCover_X, 1 , new TGeoCombiTrans(0,yPos,zPos,new TGeoRotation("r",0,0,0)));
-	if(isVert)tMagVol->AddNode(VertMagCover_X, 2 , new TGeoCombiTrans(0,-yPos,zPos,new TGeoRotation("r",0,0,0)));
-	if(isVert)tMagVol->AddNode(VertMagCover_X, 3 , new TGeoCombiTrans(0,yPos,-zPos,new TGeoRotation("r",0,0,0)));
-	if(isVert)tMagVol->AddNode(VertMagCover_X, 4 , new TGeoCombiTrans(0,-yPos,-zPos,new TGeoRotation("r",0,0,0)));
+	if(isVert)tMagVol->AddNode(VertMagCover_X, 1 , new TGeoCombiTrans(0,ypos,zpos,new TGeoRotation("r",0,0,0)));
+	if(isVert)tMagVol->AddNode(VertMagCover_X, 2 , new TGeoCombiTrans(0,-ypos,zpos,new TGeoRotation("r",0,0,0)));
+	if(isVert)tMagVol->AddNode(VertMagCover_X, 3 , new TGeoCombiTrans(0,ypos,-zpos,new TGeoRotation("r",0,0,0)));
+	if(isVert)tMagVol->AddNode(VertMagCover_X, 4 , new TGeoCombiTrans(0,-ypos,-zpos,new TGeoRotation("r",0,0,0)));
 
 
     double dxOoutCover=thiknes/2;
     double dyOoutCover=604.4*cm;
     double dzOoutCover=56.0*cm/2;
-    double xPosOutetCover = xPos-(358.2-250.4)*cm/2;
-    double yPosOutetCover = 602.4*cm;
-    double zPosOutetCover = zPos-dzOoutCover-thiknes/2;
+    double xposOutetCover = xpos-(358.2-250.4)*cm/2;
```

Comment:
What does "Outet" mean? "Oout" also is very strange. Would be good if we could give things somewhat self-explanatory names.